### PR TITLE
docs: Specify Helm chart version in "cilium install" commands

### LINKED ISSUE #26931: Make Cilium CLI installation method in the getting started guides have a `--version` like Helm

### PR DESCRIPTION
**Pull request by michi-covalent**
- For the main branch latest docs, clone the Cilium GitHub repo and use
 "--chart-directory ./install/kubernetes/cilium" flag.
- For stable branches, set "--version" flag to the version in the
  top-level VERSION file.

Fixes: #26931

Signed-off-by: Michi Mutsuzaki <michi@isovalent.com>

### CODE REVIEW

**Comment by michi-covalent** [comment]
[travis ci](https://github.com/cilium/cilium/pull/26934/checks?check_run_id=15183120726): hit https://github.com/cilium/cilium/issues/26617

**Review by meyskens — APPROVED** [review]
LGTM for me

File: Documentation/conf.py [hunk 1]
```diff
@@ -89,14 +89,17 @@
     archive_name = 'main'
     chart_release = './cilium'
     image_tag = 'latest'
+    chart_version = "--chart-directory ./install/kubernetes/cilium"
 elif branch == 'stable':
     branch = release
     archive_name = release
-    chart_release = 'cilium/cilium --version ' + release
+    chart_version = '--version' + release
```
Comment (gentoo-root) [inline Documentation/conf.py]: There used to be a space after `--version`. Won't it glue it together with the parameter value?

Comment (michi-covalent) [reply Documentation/conf.py]: good catch 🥰

Comment (michi-covalent) [reply Documentation/conf.py]: updated ✅

**Review by gentoo-root — APPROVED** [review]

**Comment by aanm** [comment]
@michi-covalent any reason why we can't backport this on all branches?

**Comment by michi-covalent** [comment]
let's do it, i just wasn't sure how far to backport 🚀

**Review by YutaroHayakawa — APPROVED** [review]
LGTM 👍

**Review by zacharysarah — APPROVED** [review]
@michi-covalent LGTM from a docs perspective. ⭐

**Comment by michi-covalent** [comment]
docs-only change. marking ready to merge

**Pull request merged**